doc/css/doc.css

```diff
@@ -40,7 +40,7 @@ div.header2 {
 }
 
 div.panel {
-  width: 640px;
```

Comment:
Without this, the reference docs for `std.manifestYamlDoc` and `std.manifestYamlStream` get rendered looking like this:

![manifestYamlDoc](https://user-images.githubusercontent.com/41255671/119733616-a4f19180-be2e-11eb-9b42-822e8650ed27.png)
![manifestYamlStream](https://user-images.githubusercontent.com/41255671/119733627-a7ec8200-be2e-11eb-9628-b7c426d62ab5.png)

From what I can gather, this happens because the lines wind up longer than the page width, because increasing the page width by adjusting this value causes them to render correctly again.  However HTML/CSS are _**far**_ from my area of expertise, so if there is a better way to correct this let me know and I will happily do that instead.